posthog/hogql_queries/experiments/experiment_trends_query_runner.py

```diff
@@ -320,7 +320,7 @@ def run(query_runner: TrendsQueryRunner, result_key: str, is_parallel: bool):
         # Statistical analysis
         control_variant, test_variants = self._get_variants_with_base_stats(count_result, exposure_result)
         if self.stats_version == 2:
-            if self.query.count_query.series[0].math:
+            if self.query.count_query.series[0].math == PropertyMathType.SUM:
```

Comment:
Can we introduce a dedicated method called `_is_trends_continuous_series()` or similar, and have a set of tests interpretation of `series`? This seems brittle at the moment.